internal/pkg/secureboot/uki/assemble.go

```diff
@@ -69,7 +69,13 @@ func (builder *Builder) assemble() error {
 
 	args = append(args, builder.SdStubPath, builder.unsignedUKIPath)
 
-	cmd := exec.Command("objcopy", args...)
+	objcopy := "/usr/x86_64-alpine-linux-musl/bin/objcopy"
```

Comment:
i hate this, but i guess let's live with this until we completely remove dependency on objcopy